views/default/admin/users/newest.php

```diff
@@ -4,6 +4,12 @@
 	'type' => 'user',
 	'subtype'=> null,
 	'full_view' => false,
+	'list_type' => 'table',
+	'cell_views' => [
```

Comment:
Why not:

```php
[
   'cells' => [
      'guid' => true, // use simple $entity->$name
      'user' => 'page/components/table/cell/default',
      'foo' => function($entity) {
          return '<p>bar</p>';
      }
   ]
]
```

For headings, iterate through granular keys combinations for entity type/subtype and find first matching key.